README.md

```diff
@@ -27,6 +27,15 @@ See also [slides of a recent talk about pyfolio.](http://nbviewer.ipython.org/fo
 
 ## Installation
 
+### (Optional) Virtual Environment
+
+You might want to use a [virtual environment](http://docs.python-guide.org/en/latest/dev/virtualenvs/) to avoid dependency conflicts between `pyfolio` and other python projects you have. To get set up with a virtual env, run:
+```bash
+
+mkvirtualenv pyfolio
```

Comment:
Is this meant for new users of pyfolio, or new developers of pyfolio?  I wouldn't expect to make a new virtualenv to use pyfolio with my existing project.